src/js/components/Header.js

```diff
@@ -34,13 +34,27 @@ class Header extends Component {
 
           <div className="navbar-links-container">
             <div className="navbar-links">
-              <a className="nav-open" href="/">Get Advice</a>
+              <a className="nav-open" href="/">
```

Comment:
"Get the App" should come before "Get Advice" like how it is in the `www.` site

Reply:
@jonuy I had it like that initially but isn't the get advice like the home button for the blog. Wouldn't it be weird if it was the second option?

Reply:
@kaladin9017 I think as a user, when you're navigating between `www.` and `advice.` and back, you don't really think of yourself as jumping between two different websites. So the swapping of the order felt a little jarring.

Also, context on where you are as a user is greatly helped by the `nav-open` style. And we should remove the `nav-open` style on the "Get the App" link.